static/js/components/VideoPlayer.js

```diff
@@ -129,38 +145,66 @@ export default class VideoPlayer extends React.Component {
     this.videoNode.style.left = `${left}px`;
     this.videoNode.style.top = `${top}px`;
     // $FlowFixMe prop does not have to be a number
-    this.videoNode.style[prop] = 'scale(2)';
+    this.videoNode.style[prop] = "scale(2)";
     this.configureCameras();
 
   }
 
   toggleFullscreen() {
-    if (isFullscreen()) {
+    let fullscreen = isFullscreen();
+    if (fullscreen) {
       // $FlowFixMe
       document[FULLSCREEN_API.exitFullscreen]();
     } else {
       // $FlowFixMe videoContainer.parentElement is not going to be null
       this.videoContainer.parentElement[FULLSCREEN_API.requestFullscreen]();
     }
+    this.player.el_.dispatchEvent(new Event(`fullscreen ${fullscreen ? "off" : "on"}`));
+  }
+
+  analyzeEvent(action: string, label: string) {
+    let percentTracked = this.percentTracked;
```

Comment:
is this `percentTracked` thingy just for tracking which values we've already tracked?

Reply:
Yes, like 10%, 20%, etc.  So it won't resend an event for 10% tracked at 20% and 30% tracked etc.

Reply:
So does that mean if I tracked to 12%, then to 22%, and then back to 13%, we'll only see two events? It seems like maybe we should record three events in that case.

Reply:
@pdpinch do you have any preference on this?

Reply:
I think I'm going to need more context.

Reply:
@pdpinch:
- User watches > 10% of the video, GA sends notification that user has watched 10%.
- User fast-forwards to 23% of the video, GA sends notification that user has watched 20%.
- User rewinds video back to 13%.  GA does not send notification that user has watched 10% again.  
- User keeps watching past 20%. GA does not send another notification that user has watched 20%.
- User keeps watching to 31%, and GA sends notification that user has watched 30% of the video.

Reply:
@pdpinch @aliceriot my opinion is that percentPlayed for any 10% increment should be recorded only once per page view, the first time it's reached. @blarghmatey talked to me about previous experiences with recording this, it can be a complex issue depending on exactly what info you want to get out of it - for example differentiating between watching the first 50% of a video vs watching 40% then FF'ing to 50%.  Maybe this should be fleshed out some more in a separate issue and future PR.

Reply:
After discussing with @pdpinch, decision for now is to stick with current functionality since it generally matches what is done with the videojs-ga plugin.  @noisecapella since Alice is on vacation would you mind finishing up the review?